protocols/cifs/src/main/java/org/opennms/netmgt/poller/monitors/JCifsMonitor.java

```diff
@@ -147,6 +148,12 @@ public boolean accept(SmbFile smbFile, String s) throws SmbException {
         // Initializing TimeoutTracker with default values
         TimeoutTracker tracker = new TimeoutTracker(parameters, DEFAULT_RETRY, DEFAULT_TIMEOUT);
 
+        try {
+            setJCifsTimeouts(tracker.getConnectionTimeout());
+        } catch (NoSuchFieldException | ClassNotFoundException | IllegalAccessException e) {
+            logger.error("Error setting JCifs timeouts ", e);
+        }
+
```

Comment:
Can't we just put the error handling in the `setJCifsTimeouts(...)` method instead? Makes it much cleaner.

Reply:
done